examples/lang/erc20/Cargo.toml

```diff
@@ -5,9 +5,13 @@ authors = ["Parity Technologies <admin@parity.io>"]
 edition = "2018"
 
 [dependencies]
-ink_core = { path = "../../../core" }
-ink_model = { path = "../../../model" }
-ink_lang = { path = "../../../lang" }
+# TODO [AJ]: remove specific branches before merging
```

Comment:
TODO!
Please resolve all TODOs and FIXMEs etc. and tell me when you are done with that.

Reply:
Yes will do that now, since we are close to merging.